src/MenuButton/index.scss

```diff
@@ -22,54 +22,32 @@
   }
 }
 
-/**
-* Reach MenuButton theme
-*/
-:root {
-  --reach-menu-button: 1;
+.nds-menubutton-popover {
+  z-index: 4;
```

Comment:
z-indexes scare me

Reply:
So everything that portals in NDS portals to `document.body`, so the order in which they get created helps set the stacking context.

Here, I'm setting 4 because the Dialog is 3